From 074e2395f81d0ecaa66b71a6c228c70b49db72e5 Mon Sep 17 00:00:00 2001
From: Andreas Gruenbacher <agruen@gnu.org>
Date: Wed, 7 Feb 2018 17:05:00 +0100
Subject: [PATCH 03/25] Test suite: fix Korn shell incompatibility

tests/merge: In a Korn shell, shift apparently fails when $# is 0.
---
 tests/merge | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/merge b/tests/merge
index b628891..e950b92 100644
--- a/tests/merge
+++ b/tests/merge
@@ -32,7 +32,7 @@ x2() {
 	shift
     done > b.sed
     echo "$body" | sed -f b.sed > b
-    shift
+    test $# -eq 0 || shift
     while test $# -gt 0 ; do
 	echo "$1"
 	shift
-- 
2.25.0

